### PR TITLE
feat: add Store schema

### PR DESCRIPTION
**Pull request by cazala**
Added `Store` schema

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint": "eslint . --ext .ts",
     "lint:fix": "eslint . --ext .ts --fix",
     "check-api": "api-extractor run --typescript-compiler-folder ./node_modules/typescript",
```
Comment (fzavalia) [inline package.json:12]: Maybe check-api would benefit from running build first such as refresh-api

File: src/dapps/store.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
+import { generateValidator, JSONSchema, ValidateFunction } from '../validation'
+
+export type Store = {
+  id: string // urn
```
Comment (fzavalia) [inline src/dapps/store.ts:4]: Not necessary for now, but it would be nice to have urns validated in the schema

Comment (fzavalia) [reply src/dapps/store.ts:4]: Just daydreaming but for example we could even add utilities like `store.id[3]` to get that section of the urn if necessary or something like a tree to navigate through different hierarchies. dunno 😂

**Review by fzavalia — APPROVED** [review]
LGTM

File: src/dapps/store.ts (hunk 2 above)
Comment (cazala) [reply src/dapps/store.ts:4]: yep, tho I dunno how to do that with json schema tbh 😅

**Pull request merged**